.github/workflows/build-ami.yml

```diff
@@ -29,18 +29,22 @@ jobs:
         runs-on: ubuntu-latest
         env:
           AWS_REGION: us-east-1
+          PACKER_VERSION: "1.10.1"
         steps:
           - name: Checkout
-            uses: actions/checkout@v3
+            uses: actions/checkout@v4
             with:
               # If the workflow is triggered manually or by schedule, uses the tag, otherwise uses the current branch  
               ref: ${{ github.event.inputs.tag || github.ref }}
           
           - name: Setup Packer
             uses: hashicorp/setup-packer@main
+            id: setup-packer
+            with:
+              version: ${{ env.PACKER_VERSION }}
```

Comment:
We can fix it here no? Or is something which changes often?

Reply:
It is the version of the packer and not the action, so it changes often. 
https://github.com/hashicorp/packer/tags. So something I will also day to fix until we run into some kind of warnings as github actions was complaining today

Reply:
@philschmid good to merge?